_specs/ecip-1089.md

```diff
@@ -0,0 +1,79 @@
+---
+lang: en
+ecip: 1089
+title: Phoenix EVM and Protocol Upgrades (Classic Option)
+status: Draft
+type: Meta
+author: /raw PONG _GHMoaCXLT (@q9f)
+created: 2020-02-24
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/262
+---
+
+### Simple Summary
+
+Enable the outstanding Ethereum Foundation _Istanbul_ network protocol upgrades on the Ethereum Classic network in a hard-fork code-named _Phoenix_ to enable maximum compatibility across these networks.
+
+This is the _classic_ option equaling to an _Istanbul_ option without gas repricing.
+
+### Abstract
+
+Add support for a subset of protocol-impacting changes introduced in the Ethereum Foundation (ETH) network via the _Istanbul_ hardforks. The proposed changes for Ethereum Classic's _Phoenix_ upgrade include:
+
+- Add Blake2 compression function `F` precompile
+- Reduce alt_bn128 precompile gas costs
+- Add ChainID opcode
+- Disable SSTORE with gasleft lower than call stipend
+- Calldata gas cost reduction
+- Add SELFBALANCE opcode
+
+This document proposes the following blocks at which to implement these changes in the Classic networks:
+
+- `999_983` on Mordor Classic PoW-testnet (April, 2020)
+- `2_200_013` on Kotti Classic PoA-testnet (April, 2020)
+- `10_500_839` on Ethereum Classic PoW-mainnet (June 10th, 2020)
+
+For more information on the opcodes and their respective EIPs and implementations, please see the _Specification_ section of this document.
+
+### Motivation
+
+To enhance the Ethereum Virtual Machine's (EVM) capabilities, various opcodes shall be added to the Ethereum Classic networks, all of which have been in use on the Ethereum Foundation networks since end of 2019.
+
+### Specification
+
+Technical specifications for each EIP can be found at those documents respectively:
+
+- [EIP-152](https://eips.ethereum.org/EIPS/eip-152): Add Blake2 compression function `F` precompile
+- [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108): Reduce alt_bn128 precompile gas costs
+- [EIP-1344](https://eips.ethereum.org/EIPS/eip-1344): Add ChainID opcode
+- [EIP-1706](https://eips.ethereum.org/EIPS/eip-1706): Disable SSTORE with gasleft lower than call stipend
```

Comment:
Is this accidentally missing EIP1283? 
(See https://github.com/ethereumclassic/ECIPs/blob/master/_specs/ecip-1078.md#specification).